presto-spark-base/src/main/java/com/facebook/presto/spark/execution/http/PagesResponse.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark.execution.http;
+
+import com.facebook.presto.spi.page.SerializedPage;
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+
+public class PagesResponse
+{
+    public static PagesResponse createPagesResponse(
+            String taskInstanceId,
+            long token,
+            long nextToken,
+            Iterable<SerializedPage> pages,
+            boolean complete)
+    {
+        return new PagesResponse(
+                taskInstanceId,
+                token,
+                nextToken,
+                pages,
+                complete);
+    }
+
+    public static PagesResponse createEmptyPagesResponse(
+            String taskInstanceId,
+            long token,
+            long nextToken,
+            boolean complete)
+    {
+        return new PagesResponse(taskInstanceId, token, nextToken,
+                ImmutableList.of(), complete);
```

Comment:
put into the same line or one param per line

Reply:
I did change my style settings to wrap... I'll change all the format thing.